protected/gatk-tools-protected/src/main/java/org/broadinstitute/gatk/tools/walkers/haplotypecaller/HaplotypeCaller.java

```diff
@@ -660,12 +661,8 @@ public void initialize() {
 
         vcfWriter.writeHeader(new VCFHeader(headerInfo, sampleSet));
 
-        try {
-            // fasta reference reader to supplement the edges of the reference sequence
-            referenceReader = new CachingIndexedFastaSequenceFile(getToolkit().getArguments().referenceFile);
-        } catch( FileNotFoundException e ) {
-            throw new UserException.CouldNotReadInputFile(getToolkit().getArguments().referenceFile, e);
-        }
+        // fasta reference reader to supplement the edges of the reference sequence
+        referenceReader = CachingIndexedFastaSequenceFile.checkAndCreate(getToolkit().getArguments().referenceFile);
```

Comment:
Because `CachingIndexedFastaSequenceFile.checkAndCreate` method catches all the exceptions which were caught here:
```
        // Read reference data by creating an IndexedFastaSequenceFile.
        try {
            return new CachingIndexedFastaSequenceFile(fastaFile);
        }
        catch (IllegalArgumentException e) {
            throw new UserException.CouldNotReadInputFile(fastaFile, "Could not read reference sequence.  The FASTA must have either a .fasta or .fa extension", e);
        }
        catch (Exception e) {
            throw new UserException.CouldNotReadInputFile(fastaFile, e);
        }
```